src/net/java/sip/communicator/impl/protocol/jabber/ProtocolProviderServiceJabberImpl.java

```diff
@@ -1746,6 +1746,11 @@ protected void initialize(EntityBareJid screenname,
                 AvatarUrl.NAMESPACE,
                 new AvatarUrl.Provider());
 
+            ProviderManager.addExtensionProvider(
+                StatsId.ELEMENT_NAME,
```

Comment:
AvatarUrl is registered above, if you want StatsId parsing, create a distinct Provider class for it